packages/flutter/test/widgets/reorderable_list_test.dart

```diff
@@ -923,6 +923,51 @@ void main() {
     ), throwsAssertionError);
   });
 
+  testWidgets('ReorderableList passes itemExtentBuilder to SliverReorderableList', (WidgetTester tester) async {
+    // Regression test for https://github.com/flutter/flutter/issues/155936
+    const int itemCount = 5;
+    const List<double> items = <double>[10.0, 20.0, 30.0, 40.0, 50.0];
+
+    void handleReorder(int fromIndex, int toIndex) {
+      if (toIndex > fromIndex) {
+        toIndex -= 1;
+      }
+      items.insert(toIndex, items.removeAt(fromIndex));
+    }
+
+    // The list has five elements, that indicate the extent for the item at the given index.
```

Comment:
I suggest removing this comment (it seems to have some grammar issues as well) and rename `items` to something more readable such as `itemExtents`.